docs/index.rst

```diff
@@ -43,80 +43,185 @@ to help you create fantastic documentation for your project.
 * **Importing your existing documentation**:
   :doc:`Import guide <intro/import-guide>`
 
+
 .. toctree::
    :maxdepth: 2
    :hidden:
-   :caption: First Steps
+   :caption: First steps
 
    intro/getting-started-with-sphinx
    intro/getting-started-with-mkdocs
+
    intro/import-guide
 
 
-.. _user-docs:
+Getting started with Read the Docs
+-----------------------------------
+
+Learn more about configuring your automated documentation builds
+and some of the core features of Read the Docs.
+
+* **Overview of core features**:
+  :doc:`features`
+
+* **Configure your documentation**:
+  :doc:`Configuration reference <config-file/index>` |
+  :doc:`webhooks` |
+  :doc:`badges` |
+  :doc:`Custom domains <custom_domains>`
+
+* **Connecting with GitHub, BitBucket, or GitLab**:
+  :doc:`Connecting your account <connected-accounts>`
+
+* **Read the Docs build and versioning process**:
+  :doc:`Build process <builds>` |
+  :doc:`Handling multiple docs versions <versions>`
+
+* **Troubleshooting**:
+  :doc:`support` |
+  :doc:`Frequently asked questions <faq>`
 
 .. toctree::
-   :maxdepth: 2
-   :caption: User Documentation
+   :maxdepth: 1
+   :hidden:
+   :caption: Getting started
 
-   versions
-   builds
    features
+
+   config-file/index
+   webhooks
+   badges
+   custom_domains
+
    connected-accounts
+
+   builds
+   versions
+
    support
    faq
-   config-file/index
-   guides/index
-   api/index
-   embed
 
-.. _feature-docs:
+
+Advanced features of Read the Docs
+----------------------------------
+
+Read the Docs offers many advanced features and options.
+Learn more about these integrations and how you can get the most
+out of your documentation and Read the Docs.
+
+* **Advanced project configuration**:
+  :doc:`subprojects` |
+  :doc:`Single version docs <single_version>` |
+  :doc:`Privacy levels <privacy>`
+
+* **Multi-language documentation**:
+  :doc:`Translations and localization <localization>`
+
+* **Redirects**:
+  :doc:`User defined redirects <user-defined-redirects>` |
+  :doc:`Automatic redirects <automatic-redirects>`
+
+* **Topic specific guides**:
+  :doc:`How-to guides <guides/index>`
```

Comment:
What about putting this in the Troubleshooting section?

Reply:
These guides (eg. "managing translations" or "adding custom JS/CSS to sphinx") are not exactly troubleshooting. That's not to say we can't come up with something better than "how-to guides".

Maybe we could just have a full sentence giving more details about the guides. Django does something like that in their documentation:

![Screen Shot 2019-06-17 at 4 45 12 PM](https://user-images.githubusercontent.com/185043/59643903-72f23380-911f-11e9-825c-571ee8bcde5a.png)

Travis has a whole section on their homepage for "Language specific guides":

![Screen Shot 2019-06-17 at 4 44 57 PM](https://user-images.githubusercontent.com/185043/59643917-82717c80-911f-11e9-9a61-0726ca391b84.png)

Our guides are a little too varied for what Travis does but maybe something similar to what Django does.